src/metatensor/models/utils/merge_capabilities.py

```diff
@@ -4,26 +4,29 @@
 
 
 def merge_capabilities(
```

Comment:
I'm not sure I understand why this function is needed now. Could you explain?

Reply:
When a pre-trained model is fine-tuned on a different dataset, that function takes the capabilities of the old model and merges it with the capabilities generated from the new dataset

Reply:
Yes, that would be the idea (for SOAP-BPNN and the Alchemical Model, at least). Many people have successfully pre-trained on cheap DFT and fine-tuned on coupled cluster. In that case, the model gains a capability

Reply:
I think it does, you can now predict with the base model and with the new "head" for coupled cluster. You don't mix the two, you just use the model's internal representation to learn the new property

Reply:
Ok, fair point. But this seems rater specific to a given architecture. Let's keep it for now and revisit later!